src/scripts/Deploy.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {DSTest} from "ds-test/src/test.sol";
+import {Hevm} from "solmate/test/utils/Hevm.sol";
+
+import "forge-std/src/Script.sol";
+
+import {AludelFactory} from "../contracts/AludelFactory.sol";
+
+import {IAludel} from "../contracts/aludel/IAludel.sol";
+import {Aludel} from "../contracts/aludel/Aludel.sol";
+import {AludelV1} from "../contracts/aludel/legacy/AludelV1.sol";
+import {GeyserV2} from "../contracts/aludel/legacy/GeyserV2.sol";
+
+contract DeployFactory is Script {
+
+    function run() external {
+        
+        // default value: 
+        address recipient = vm.envAddress("ALUDEL_FACTORY_RECIPIENT");
+
+        // default value: 100
+        uint16 bps = uint16(vm.envUint("ALUDEL_FACTORY_BPS"));
+
+        vm.startBroadcast();
+        AludelFactory factory = new AludelFactory(recipient, bps);
+        vm.stopBroadcast();
+
+        // AludelV1 deployment
```

Comment:
This kind of comment isn't really helpful, the _what_ and _how_ of the lines below is obvious reading them, however, the _why_ (deploy an empty contract so pre-existing aludels can be set to something) isn't , so it'd be worth commenting that instead.

Also, consider just creating an EmptyContract or something more explicit, instead of using a contract & path name which suggests a real contract is being used.